datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -164,4 +189,26 @@ private FlowIdentifier notify(String selector, SdxEvent acceptable) {
         }
 
     }
+
+    /**
+     * Get the backup location.
+     * @param envCrn Environemnt CRN.
+     * @return backuplocation configured for the environment, If not, returns the log location.
+     */
+    String getBackupLocation(String envCrn) {
```

Comment:
this must be private

Reply:
I made the change protected so that tests can use it.

Reply:
only public method should be tested. no access should be changed because of testing. if it's hard to test something then it needs refactoring like moving into a separate class.